### PR TITLE
Updated to enable twitter client side login

### PR DESCRIPTION
**Pull request by mamaso**

### CODE REVIEW

File: e2etest/tables/authenticated.js [hunk 1]
```diff
@@ -37,4 +38,16 @@ function matchUserId(id) {
...
+
+function extractClientTokens(identities) {
+	return Object.keys(identities).reduce(function (tokens, provider) {
+		tokens[provider] = {
+			access_token: identities[provider].access_token;
+		};
+		
+		if (provider === 'twitter') {
+			tokens[provider].access_token_secret = identities[provider].access_token_secret;
+		}
```
Comment (danderson00) [inline e2etest/tables/authenticated.js]: I think you need to return tokens here

Comment (mamaso) [inline e2etest/tables/authenticated.js]: maybe a misnomer:

if you look at .net, it returns an object like 
{
  facebook: { access_token: ... },
  google: ..... etc
}

On the client side, platforms select the provider from the lastUserIdentityObject, which then contains the relevant token

Comment (danderson00) [inline e2etest/tables/authenticated.js]: I just mean that you need to return the tokens variable from the reduce function, or the reduce won't work.

Comment (mamaso) [inline e2etest/tables/authenticated.js]: Ah yep my bad, had this working on the server but failed the copy paste :/

**Pull request merged**